Automattic/jetpack
# Videopress: disable uploading for Free WoA

This PR disables uploading video files for WoA sites with a Free plan.

#### Changes proposed in this Pull Request:

* videopress: disable uploading video files for WoA with Free plan.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Test with a WoA site with a Free plan
* Go to edit a post
* Add a video block
* check the uploading buttons are disabled 

before | after
----|-----
![image](https://user-images.githubusercontent.com/77539/119545475-3d254300-bd69-11eb-8916-39c5ba5350b8.png) | ![image](https://user-images.githubusercontent.com/77539/119548265-66939e00-bd6c-11eb-958d-fe87b681aa01.png)
